.github/workflows/continuous-integration.yml

```diff
@@ -66,9 +66,9 @@ jobs:
         env:
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
         with:
-          arguments: jacocoTestReport coverallsJacoco
+          arguments: codeCoverageReport coveralls
         continue-on-error: true
-        if: runner.os == 'Linux' && matrix.java == '8' && matrix.epVersion == '2.4.0' && github.repository == 'uber/NullAway'
+        if: runner.os == 'Linux' && matrix.java == '11' && matrix.epVersion == '2.4.0' && github.repository == 'uber/NullAway'
```

Comment:
Need this change since computing the overall code coverage report does not work on Java 8 (due to some incompatibility with the JDK 17 tests)